docs/expression.md

```diff
@@ -133,3 +139,27 @@ FunctionExpression[] functionExpressions =
   </TabItem>
 </Tabs>
 
+## Unparsed expression
+
+An unparsed expression, a special type of expression, currently serves exclusively to represent the default value of a column when it's unsolvable.
+The following shows the data structure of an unparsed expression in JSON and Java, enabling easy retrieval of its value.
+
+<Tabs>
+  <TabItem value="Json" label="Json">
+
+```json
+{
+  "type": "unparsed",
+  "unparsedExpression": "(curdate() + interval 1 year)"
+}
+```
+
+  </TabItem>
+  <TabItem value="java" label="Java">
+
+```java
+String unparsedValue = ((UnparsedExpression) expressino).unparsedExpression();
```

Comment:
Can you make the Java and JSON code consistent?

Reply:
This code shows how to retrieve from a `UnparsedExpression`.  I will add a comment to the code.